mono/utils/mono-dl-fallback.h

```diff
@@ -10,10 +10,11 @@
 MONO_BEGIN_DECLS
 
 enum {
-	MONO_DL_EAGER = 0,
-	MONO_DL_LAZY  = 1,
-	MONO_DL_LOCAL = 2,
-	MONO_DL_MASK  = 3
+	MONO_DL_EAGER  = 0,
+	MONO_DL_LAZY   = 1,
+	MONO_DL_LOCAL  = 2,
+	MONO_DL_MASK   = 3,
```

Comment:
Maybe comment what `mask` is. I'm left wanting to review the code, to see if it should now be `7`, or if it is even used.

Reply:
Appears to be unused. Consider removing. Maybe it is a random number in the embedding API.
Removing it will break embedders:
```
#include "mono.h"
#include <stdio.h>
int main()
{
 printfi"%d\n", MONO_DL_MASK);
}
```

Reply:
It's not used anywhere in the runtime and I have no clue why it exists. I guess someone wanted a convenient way to set LOCAL and LAZY?

Reply:
Yeah, I don't think I can remove it since it's public, so it'll just continue to exist with no real purpose.

Reply:
Maybe adding a comment.
Ideally it'd be marked "external only" but I've never seen that for enums/defines at the language level.
So maybe just a comment.

Reply:
Oh I can't read, it was a mask. But well, it's not used internally and is no longer a full mask since I'm using the next bit, so I'll just comment not to use it. Pity I can't change or remove it, but it is what it is.